lib/vdsm/tool/configurators/multipath.py

```diff
@@ -253,9 +253,14 @@
 #   blacklist_exceptions {
 #       protocol "(scsi:spi|scsi:ssa)"
 #   }
+#
+# We blacklist all RADOS Block Device (RBD) devices.
+# When using Ceph, multipath prevents the rbd devices from being unmapped
+# and the devices remain busy.
 
 blacklist {
     protocol "(scsi:adt|scsi:sbp)"
+    devnode "^(rbd)[0-9]*"
```

Comment:
This change will not be effective on upgrades without creating a new revision
of the configuration. You have to change this:

```
_CURRENT_TAG = "# VDSM REVISION 2.0"
_OLD_TAGS = (
    "# VDSM REVISION 1.9",
```

The current revision must be 2.1, and revision 2.0
must move to the old tags section.

Please check previous changes in this file, they show how to change this
file. One example is d2dcd5d015463589ea9aad147b75c95d74effa0f.

Reply:
Thanks for the example.
ack